Clients/src/presentation/components/RichTextEditor/index.tsx

```
@@ -0,0 +1,147 @@
+      case "bold":
+        editor.chain().focus().toggleBold().run();
+        break;
+      case "italic":
+        editor.chain().focus().toggleItalic().run();
+        break;
+      case "uppercase":
+        const uppercaseText = editor.getText().toUpperCase();
+        editor.commands.setContent(uppercaseText);
+        break;
+      case "lowercase":
+        const lowercaseText = editor.getText().toLowerCase();
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)